devito/core/cpu.py

```diff
@@ -322,7 +323,8 @@ def _make_clusters_passes_mapper(cls, **kwargs):
             'cire-divs': lambda i: cire(i, 'divs', sregistry, options, platform),
             'cse': lambda i: cse(i, sregistry),
             'opt-pows': optimize_pows,
-            'topofuse': lambda i: fuse(i, toposort=True)
+            'topofuse': lambda i: fuse(i, toposort=True),
+            'skewing': skewing
```

Comment:
So probably not a fun question, but is that the "best" order? I.e does applying skewing earlier in the passes change the output?

Reply:
Before submitting this PR, I incorporated the pass in the advanced mode, looking at two cases
a) applied, just after blocking
b) applied as a last pass

AFAIR there was no diff in the resulting operators' OI. Will need to pay more attention though.Since the only parameter that may affect skewing from a kernel point of view is the space order, I think it will make no diff.

As a said, will pay more attention later on this.